Bookish/Views/Home/Index.cshtml

```diff
@@ -4,34 +4,80 @@
+                    </tr>
+                    <tr>
+                        <td><b>Book publisher : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.Publisher, new { @type = "text", placeholder = "Publisher"})</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book publish date : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.DatePublished, new { @type = "date", placeholder = "Publish date", required = "required" })</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book author forename : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.AuthorForename, new { @type = "text", placeholder = "Forename", pattern = "^[A-Za-z ,.'-]+$"})</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book author surname : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.AuthorSurname, new { @type = "text", placeholder = "Surname", pattern = "^[A-Za-z ,.'-]+$" })</td>
+                    </tr>
+                    <tr>
+                        <td><b>Number of copies : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.NewCopies, new { @type = "number", placeholder = "copies", required = "required" })</td>
+                    </tr>
+                </tbody>
+            </table>
+            <br/><input type="submit" value="Enter book info"/>
         </fieldset>
     }
-    
+</div>
+<div id="Enter a User" class="tabcontent" style="display:none;">
     @using (Html.BeginForm("AddNewBorrower", "Catalog", FormMethod.Post))
-        {
-            <fieldset>
-                
-                <legend>Enter a borrower:</legend>
-                @Html.TextBoxFor(m => m.BorrowerInput.PhoneNumber, new{@type="text", placeholder="Phone number", pattern="^[\\+]?[(]?[0-9]{3}[)]?[-\\s\\.]?[0-9]{3}[-\\s\\.]?[0-9]{4,6}$", title = "Please enter a phone number"})
-                @Html.TextBoxFor(m => m.BorrowerInput.Forename, new{@type="text", placeholder="Forename", pattern="^[A-Za-z ,.'-]+$" })
-                @Html.TextBoxFor(m => m.BorrowerInput.Surname, new{@type="text", placeholder="Surname", pattern="^[A-Za-z ,.'-]+$" })
-                <input type="submit" value="Enter borrower info" />
-    
-            </fieldset>
-        }
+    {
+        <fieldset>
+            <legend>Enter a user:</legend>
+            <table class="table table-hover table-sm">
+
+                <tbody>
+                    <tr>
+                        <td><b>User phone number : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BorrowerInput.PhoneNumber, new{@type="text", placeholder="Phone number", pattern="^[\\+]?[(]?[0-9]{3}[)]?[-\\s\\.]?[0-9]{3}[-\\s\\.]?[0-9]{4,6}$", title = "Please enter a phone number", required = "required"})</td>

```

Comment:
You should nicely space the `new { @type = "text"...` stuff like you have done earlier in the file